code/game/machinery/alarm.dm

```diff
@@ -939,6 +932,8 @@ FIRE ALARM
 
 	if (istype(W, /obj/item/weapon/screwdriver) && buildstage == 2)

```

Comment:
This should really be using the standards in screwToggle, but it's not entirely a job for a hotfix PR.